README.md

```diff
@@ -1,2 +1,17 @@
-Hello, world!
-its me Margaret )orreally jsut Sheila
+# OPRE OPS
+
+## Getting Started
+### Running the Application locally
+
+Application built using [pack Buildpack CLI](https://github.com/buildpacks/pack) and runs within Docker.
+
+Instructions to start app locally:
+
+1. Make sure you have Docker installed locally
```

Comment:
installed and *started* locally.  I ran into an issue because I didn't start Docker.